src/main/java/controller/EditProductServlet.java

```diff
@@ -13,7 +13,7 @@
 import java.io.IOException;
 import java.util.Optional;
 
-@WebServlet(value = "/edit/product")
+@WebServlet(value = "/admin/edit/product")
 public class EditProductServlet extends HttpServlet {
 
     private static final ProductService productService = ProductServiceFactory.getInstance();
```

Comment:
in method post() better call method update which will edit our product
the same with users

Reply:
This method was accepted in HW22, i didn't change nothing except adding "/admin". So I will change it in homework with database